public/app/core/services/context_srv.ts

```diff
@@ -180,23 +164,14 @@ export class ContextSrv {
   }
 
   hasAccessToExplore() {
-    if (this.accessControlEnabled()) {
-      return this.hasPermission(AccessControlAction.DataSourcesExplore) && config.exploreEnabled;
-    }
-    return (this.isEditor || config.viewersCanEdit) && config.exploreEnabled;
+    return this.hasPermission(AccessControlAction.DataSourcesExplore) && config.exploreEnabled;
   }
 
   hasAccess(action: string, fallBack: boolean): boolean {
```

Comment:
Can you please make a note to remove this and `hasAccessInMetadata` altogether (should be replaced by `hasPermission` and `hasPermissionInMetadata`)?

Reply:
tracked in https://github.com/grafana/grafana/issues/74384